support/oidc-discovery-provider/main.go

```diff
@@ -41,11 +42,22 @@ func run(configPath string) error {
 	var source JWKSSource
 	switch {
 	case config.RegistrationAPI != nil:
-		source, err = NewRegistrationAPISource(RegistrationAPISourceConfig{
+		log.Warn("The registration_api configuration is deprecated in favor of server_api; please update your configuration")
+		address, err := addressFromSocketPath(config.RegistrationAPI.SocketPath)
+		if err != nil {
+			return err
+		}
+		source, err = NewServerAPISource(ServerAPISourceConfig{
```

Comment:
Since this `err` variable is a new one created from the `address, err := ` assignment, this `err` needs to be checked separately.

Reply:
Oh yep, thanks for catching the aliasing... will fix.